site/src/components/page-faq/contact-modal.scss

```diff
@@ -0,0 +1,37 @@
+.modal-wrapper {
+    position: fixed;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    display: none;
+    z-index: 1;
+  }
+  .visible {
+    background-color: rgba(0,0,0,0.4);
+    visibility: visible;
+    display: inherit;
+  }
+  .modal {
+    padding: 40px 40px 40px 40px;
+    background-color: #1C1834;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    border-radius: 2px;
+    min-width: 300px;
+    z-index: 10;
+    @include on-mobile() {
```

Comment:
I'm having issues scrolling at mobile resolution. If it's not just me, maybe use a grid or flexbox to position the modal inside the wrapper?

Reply:
added scrolling to smaller mobile devices, worked on iPhone x but not on smaller phones.